packages/event-consumer/src/utilities/config.ts

```diff
@@ -22,15 +22,31 @@ const KafkaConfig = z
     KAFKA_BROKERS: z.string(),
     KAFKA_CLIENT_ID: z.string(),
     KAFKA_GROUP_ID: z.string(),
+    KAFKA_DISABLE_AWS_IAM_AUTH: z
+      .enum(["true", "false", "TRUE", "FALSE"])
+      .optional(),
   })
   .transform((c) => ({
     kafkaBrokers: c.KAFKA_BROKERS,
     kafkaClientId: c.KAFKA_CLIENT_ID,
     kafkaGroupId: c.KAFKA_GROUP_ID,
+    kafkaDisableAwsIamAuth:
+      c.KAFKA_DISABLE_AWS_IAM_AUTH === "TRUE" ||
+      c.KAFKA_DISABLE_AWS_IAM_AUTH === "true",
   }));
 export type KafkaConfig = z.infer<typeof KafkaConfig>;
 
-export const Config = MongoConfig.and(KafkaConfig);
+const AWSConfig = z
+  .object({
+    AWS_REGION: z.string(),
```

Comment:
should the region be obtained from AWS "context" like credentials, instead of only env vars?

Reply:
makes sense, thanks